README.md

```diff
@@ -113,6 +113,30 @@ The `--fix` option on the command line automatically fixes problems reported by
 
 <!--RULES_TABLE_END-->
 
+## :couple: FAQ
+
+### What is "Use the latest vue-eslint-parser" errors?
+
+The most rules of `eslint-plugin-vue` requires `vue-eslint-parser` to check `<template>` ASTs.
+
+Please add one of the following settings into your configuration:
```

Comment:
What do you think about: `Make sure you have one of the following settings in your **.eslintrc.js**:` ?